esmvaltool/diag_scripts/autoassess/land_surface_soilmoisture/soilmoisture.py

```diff
@@ -5,37 +5,58 @@
 import numpy as np
 import iris
 from esmvalcore.preprocessor import regrid
+from esmvaltool.diag_scripts.shared._base import ProvenanceLogger
 from esmvaltool.diag_scripts.shared._supermeans import get_supermean
 
-
 logger = logging.getLogger(__name__)
 
 
+def get_provenance_record(plot_file, caption, run):
+    """Create a provenance record describing the diagnostic data and plot."""
+    record = {
+        'caption': caption,
+        'statistics': ['mean'],
+        'domains': ['global'],
+        'plot_type': 'map',
```

Comment:
soilmoisture diagnostic only produces the metrics plot, so 'map' isn't right. Not sure how best to describe the metrics plot, perhaps 'point', or 'bar'? Is there a list of allowed values for 'plot_type'?

Reply:
we can probably put `autoassess metric`? AFAIK there is no restricted list of possible values here